Jenkinsfile

```diff
@@ -125,6 +125,13 @@ pipeline {
   }
 
   post {
+    success {
```

Comment:
I think this should only notify if this is a deploy (master / production-release job), https://github.com/zooniverse/panoptes/blob/4886b71d1857afbc1a5970d122b4a256bf550819/Jenkinsfile#L141-L151